src/components/can_cooperation/src/commands/get_interior_vehicle_data_request.cc

```diff
@@ -70,16 +72,17 @@ void GetInteriorVehicleDataRequest::OnEvent(
       (functional_modules::hmi_api::get_interior_vehicle_data == event.id()));
 
   application_manager::Message& hmi_response = *(event.event_message());
-  const bool validate_result = service()->ValidateMessageBySchema(hmi_response);
+  const bool validate_result = application_manager::SUCCESS ==
+                               service()->ValidateMessageBySchema(hmi_response);
+  LOG4CXX_DEBUG(logger_,
+                "HMI response validation result is " << validate_result);
   const Json::Value value =
       MessageHelper::StringToValue(hmi_response.json_message());
   std::string result_code;
   std::string info;
   bool success = validate_result && ParseResultCode(value, result_code, info);
 
-  const bool is_generic_error =
-      (!validate_result) || (success && false == CheckForRequestedData(value));
```

Comment:
@AByzhynar why do this checking was removed? Is it enough to validate by schema?